### PR TITLE
schedule commands timing, retry

### PR DESCRIPTION
**Pull request by MichaelDvP**
I've changed the timings: timer commands to esp-uptime, calender to RTC. Also added a retry counter, we can define if we like a retry for startup commands. Still wonder for what these startup commands are usefull, but i know `<device>/<cmd>` will not work before the bus is scanned.

### CODE REVIEW

**Comment by proddy** [comment]
I don't personally see the need for Boot up either. The use case came from https://github.com/emsesp/EMS-ESP32/discussions/1065

**Comment by proddy** [comment]
I closed #127 because there was no interest from the original author for 15 months

File: src/web/WebSchedulerService.cpp [hunk 1]
```diff
@@ -84,75 +83,107 @@ StateUpdateResult WebScheduler::update(JsonObject & root, WebScheduler & webSche
...
+        for (ScheduleItem & scheduleItem : *scheduleItems) {
+            if (scheduleItem.active && scheduleItem.flags == SCHEDULEFLAG_SCHEDULE_TIMER && scheduleItem.elapsed_min == 0) {
+                scheduleItem.retry_cnt = command(scheduleItem.cmd.c_str(), scheduleItem.value.c_str()) ? 0xFF : 0;
+            }
+        }
+        last_tm_min = 0; // startup done, now use for RTC
+    }
+    // check timer every minute, sync to EMS-ESP clock
+    uint32_t uptime_min = uuid::get_uptime_sec() / 60;
+    if (last_uptime_min != uptime_min || last_uptime_min == -1) {
```
Comment (proddy) [inline src/web/WebSchedulerService.cpp:157]: last_uptime_min is unsigned so this will always be false

Comment (MichaelDvP) [reply src/web/WebSchedulerService.cpp:157]: Ups, yes, can be removed, left from a first try to put startup and timer in one for().

**Pull request merged**